.hashibot.hcl

```diff
@@ -35,6 +35,30 @@ behavior "deprecated_import_commenter" "hashicorp_terraform" {
   EOF
 }
 
+behavior "deprecated_import_commenter" "hashicorp_terraform" {
+  import_regexp = "github.com/hashicorp/terraform-plugin-sdk/helper/hashcode"
+  marker_label  = "terraform-plugin-sdk-migration"
```

Comment:
I don't know what `marker_label` means, so I left this as it was.
Can you check this for me please? @bflad